docs/content/users/quickstart.md

```diff
@@ -587,6 +614,57 @@ The Laravel project type can be used for [Statamic](https://statamic.com/) like
     ddev launch
     ```
 
+## Symfony
+
+There are many ways to install Symfony, here are a few of them based on the [Symfony docs](https://symfony.com/doc/current/setup.html).
+
+If your project uses a database you'll want to set the [DB connection string](https://symfony.com/doc/current/doctrine.html#configuring-the-database) in the `.env`. If using the default MariaDB configuration, you'll want `DATABASE_URL="mysql://db:db@db:3306/db?serverVersion=10.11"`. If you're using a different database type or version, see `ddev describe` for the type and version.
+
+=== "Composer"
+
+    ```bash
+    mkdir my-symfony && cd my-symfony
+    ddev config --docroot=public
+    ddev composer create symfony/skeleton:"7.0.*"
+    ddev composer require webapp
+    # When it asks if you want to include docker configuration, say "no" with "x"
+    ddev launch
+    ```
+
+=== "Symfony CLI"
+
+    In a future release the Symfony CLI will be provided by default in `ddev-webserver`, but for now it needs to be configured.
+
+    * Create a `.ddev/web-build/Dockerfile.symfony-cli` with the contents:
+
+    ```bash
+    RUN curl -1sLf 'https://dl.cloudsmith.io/public/symfony/stable/setup.deb.sh' | sudo -E bash
+    RUN sudo apt install -y symfony-cli
+    ```
+
+    Then:
+
+    ```bash
+    mkdir my-symfony && cd my-symfony
+    ddev config --docroot=public
```

Comment:
We can use `echo` to add the file, as in other quickstarts:

````suggestion
    ```bash
    mkdir my-symfony && cd my-symfony
    ddev config --docroot=public
    echo "RUN curl -1sLf 'https://dl.cloudsmith.io/public/symfony/stable/setup.deb.sh' | sudo -E bash
    RUN sudo apt install -y symfony-cli" >.ddev/web-build/Dockerfile.symfony-cli
````

Reply:
Why this? Is  this better?

Reply:
Because the order of actions is wrong:

The example starts by creating `.ddev/web-build/Dockerfile.symfony-cli` and then you create a project folder
```
mkdir my-symfony && cd my-symfony
ddev config --docroot=public
```

Reply:
Make sense.